lib/internal/url.js

```diff
@@ -263,6 +256,48 @@ class URL {
     ret += '}';
     return ret;
   }
+
+  [kFormat](options) {
```

Comment:
Putting the symbols here at all (enumerable or not) creates an observable difference from the spec via `Object.getOwnPropertySymbols`. But I guess there are several of those already (special/cannotBeABase) so IMO it's probably not worth changing...